### PR TITLE
Microverse HTML/CSS Project Request

### PR DESCRIPTION
**Pull request by katarighe**
This is the project that I completed through HTML, CSS, Visual Studio Code and Git Bash. Done in less than 1 hour.

### CODE REVIEW

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @katarighe,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Great project setup.✔️
- Added linter correctly. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments below._

- [x] Hey, please, kindly use this [readme template](https://github.com/microverseinc/readme-template), and customize it to suit the project needs to have a professional readme in you repository.



## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Strangeal) [inline index.html:1]: - [ ] As suggested by the reviewer, Kindly consider documenting your project professionally using the [README](https://github.com/microverseinc/readme-template) template provided in the project requirement.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @katarighe, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Eternalgratis — APPROVED** [review]
# Status: Approved 🟢 
Hi @katarighe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great implementation of all the requirements in this milestone. 👏
 
image

## Highlights
- No Linter error ✅
- Descriptive PR summary ✅

# Optional suggestions
I noticed your README isn't in your feature branch which means your GitHub flow is incorrect.  Please update your feature branch so it has your descriptive README as it is a crucial part of this project and also a professional way of using GitHub. I'm making it optional because the previous reviewer omitted it. Kindly make the necessary changes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**